docs/source/cli.rst

```diff
@@ -36,7 +36,7 @@ create the environment file for you. Below are the currently (known) plugins, se
     * - Organization
       - Link
     * - Utrecht University
-      - https://github.com/UtrechtUniversity/ibridges-servers-uu
+      - https://github.com/iBridges-for-iRODS/ibridges-servers-uu
```

Comment:
Do you want to move them here too, or shall they stay under the UU organisation?

Reply:
I guess it makes more sense to have in under UtrechtUniversity, though I just like the idea of having everything under the same organization.